doc/classes/RibbonTrailMesh.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="RibbonTrailMesh" inherits="PrimitiveMesh" version="4.0">
+	<brief_description>
+	</brief_description>
+	<description>
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+	</methods>
+	<members>
+		<member name="curve" type="Curve" setter="set_curve" getter="get_curve">
+		</member>
+		<member name="section_length" type="float" setter="set_section_length" getter="get_section_length" default="0.2">
+		</member>
+		<member name="section_segments" type="int" setter="set_section_segments" getter="get_section_segments" default="3">
+		</member>
+		<member name="sections" type="int" setter="set_sections" getter="get_sections" default="5">
+		</member>
+		<member name="shape" type="int" setter="set_shape" getter="get_shape" enum="RibbonTrailMesh.Shape" default="1">
```

Comment:
You need to bind this enum with `VARIANT_ENUM_CAST` in the header where it's defined, and bind the enum's constants in `_bind_method`.

Reply:
`RibbonTrailMesh.Shape` is already bound:
https://github.com/godotengine/godot/blob/dea84c9d28efff49054d3ecb2e5f119982cf5845/scene/resources/primitive_meshes.h#L432
Its constants are not, so I'll add those.